content/_data/changelogs/lts.yml

```diff
@@ -6239,6 +6239,362 @@
     message: |-
       Correction of Label expression including a "implies" relationship without spaces around.
 
+- version: "2.319.1"
+  date: 2021-08-25
+  lts_predecessor: "2.303.3"
+  lts_baseline: "2.319"
+  changes: # compared to lts_baseline 2.319 - extracted from the RC commit(s)
+
+  - type: bug
+    category: bug
+    pull: 5900
+    issue: 67063
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-67063] Revert canonicalization of entries when untarring"
+    message: |-
+      Do not attempt to canonicalize tar entries when untaring, as the result may be unexpected for symlinks.
+
+  - type: bug
+    category: bug
+    pull: 5881
+    issue: 67061
+    authors:
+      - daniel-beck
+    references:
+      - pull: 5881
+      - issue: 67061
+      - url: /doc/upgrade-guide/2.303/#SECURITY-2455
+        title: Upgrade guide - Agent to controller path filter security fixes
+    pr_title: "[JENKINS-67061] Fix overload in ConfigDirectory"
+    message: |-
+      Fix form submission for file access rules of agent to controller security subsystem (regression in 2.111).
+
+  - type: bug
+    category: bug
+    pull: 5875
+    issue: 67028
+    authors:
+      - NotMyFault
+    pr_title: "[JENKINS-67028] Fix missing project hyperlink in build history"
+    message: |-
+      Fix missing hyperlink in build history (regression in 2.314).
+
+  - type: bug
+    category: bug
+    pull: 5864
+    issue: 67002
+    authors:
+      - jtnord
+    pr_title: "[JENKINS-67002] protect Jenkins against bad RestartListeners"
+    message: |-
+      An exception thrown by a <code>RestartListener</code> no longer leaves Jenkins in a zombie-like state.
+
+  - type: bug
+    category: regression
+    pull: 5865
+    issue: 66993
+    authors:
+      - basil
+    pr_title: "[JENKINS-66993] Do not register `AntClassLoader` as parallel-capable"
+    message: |-
+      Prevent <code>LinkageError</code> during class loading (regression in 2.309).
+
+  - type: bug
+    category: bug
+    pull: 5906
+    issue: 66969
+    authors:
+      - janfaracik
+      - timja
+    pr_title: "[JENKINS-66969] Fix ongoing first build from disappearing from\
+      \ build history"
+    message: |-
+      Show ongoing first build in build history (regression in 2.314).
+
+  lts_changes: # compared to lts_predecessor 2.303.3 (selected by personal review)
+
+  # Inclusive naming improvements
+
+  - type: major rfe
```

Comment:
Do we really need 4 changelog entries for this?

Reply:
I liked the separate items in the changelog because I felt they were distinct enough to justify separate entries.  They were in the weekly release as 4 items.  I think we could combine the 4 entries into 2 entries if that addresses your concern.  The two entries would be:

* Replace the term "master" with "controller" (for the main Jenkins application) or "built-in node" in user interface strings and documentation as appropriate. Warn about use of `master` in a label expression when that's no longer in use. (jenkinsci/jenkins#5425, jenkinsci/jenkins#5674)
* Add migration code to change the node name (e.g. <code>NODE_NAME</code> environment variable) and label of the built-in node only after explicit migration by an administrator. New installations get the new node and label immediately. If a job definition, Pipeline definition, or tool installer reference must be tied to the built-in node, it should use the label "`built-in`". Add the system property `jenkins.model.Jenkins.nodeNameAndSelfLabelOverride` to specify a different node name and label for the built-in node (e.g. for Configuration as Code use cases) than the one otherwise determined.

Would that address your concern @timja?

@daniel-beck any guidance from you?

Reply:
From what I understand weekly changelog is normally squashed down for LTS, just because changes were delivered separately in weekly doesn't mean they should be kept separate for LTS.

> Warn about use of master in a label expression when that's no longer in use.
Does this need an entry?

> Add migration code to change the node name (e.g. NODE_NAME environment variable) and label of the built-in node only after explicit migration by an administrator. New installations get the new node and label immediately.

talking about adding code in a changelog seems a bit weird, button or feature maybe? but I'm not liking it a lot.

> Add the system property jenkins.model.Jenkins.nodeNameAndSelfLabelOverride to specify a different node name and label for the built-in node (e.g. for Configuration as Code use cases) than the one otherwise determined

Does this warrant being in the changelog or in the upgrade guide?

Reply:
We reviewed in Docs Office Hours.  @StackScribe and @kwhetstone agreed with your observation.  We've made the changes you recommended.  Thanks!